### PR TITLE
Feature: Pending message on join

### LINKED ISSUE #2: [Enhancement] Configuration option to notify Staff of any open tickets
Would be awesome for when Staff members join and there's an existing ticket queue that hasn't been fully taken care of. :)

### PR DESCRIPTION
**Pull request by Dartanboy**
Admins get a message when they join the server informing them of pending tickets.

### CODE REVIEW

**Comment by denniemok** [comment]
Hi @Dartanboy ! Thanks for contributing!

I spotted a potential defects that may require some changes.

When a player with the admin permission joins the server, the plugin adds that player to the Admin list, so that they will receive admin broadcasts such as when there is a newly created tickets and so on.

The issue for the edit is that when `message_pending_on_join` is false, then admins will not be added to the admin list. Sadly, they will no longer receive any admin broadcasts because of that. A potential fix is to move the condition down a bit to account for such scenario.

Otherwise, looks good to me! Again, thanks for your great work! Let me know after you have made the modification so that I can merge into main branch and publish new builds for public.

Best wishes and have a nice day~

File: src/main/java/com/battleasya/admin360/listener/JoinLeaveEvent.java [hunk 1]
```diff
@@ -20,10 +21,11 @@ public void onJoin(PlayerJoinEvent event){
 
         Player player = event.getPlayer();
 
-        if (User.hasPermission(player, Permission.ATTEND_TICKET, false)){
+        if (Config.message_pending_on_join &&
+                User.hasPermission(player, Permission.ATTEND_TICKET, false)){
             Admin.addAdmin(player.getUniqueId());
+            User.messagePlayer(player, Config.pending_tickets.replace("{num}", String.valueOf(Request.requestPending.size())));
         }
```
Comment (denniemok) [inline src/main/java/com/battleasya/admin360/listener/JoinLeaveEvent.java:30]: Suggested changes:
```
if (User.hasPermission(player, Permission.ATTEND_TICKET, false)){
    Admin.addAdmin(player.getUniqueId());
    if (Config.message_pending_on_join) {
       	User.messagePlayer(player, Config.pending_tickets.replace(
       	 "<PENDING>", Integer.toString(Request.getPndLstSize())
       	));
    }
}
```
- [Critical] When `message_pending_on_join` is false, admins will not be added to the admin list, depriving them from receiving any admin broadcasts.

- [Optional] Utilise the available class function `Request.getPndLstSize()`.

- [Optional] Use `<PENDING>` as the message placeholder to better align with the default config style.

**Review by denniemok — CHANGES_REQUESTED** [review]
Required changes to `JoinLeaveEvent.java`

File: src/main/java/com/battleasya/admin360/listener/JoinLeaveEvent.java (hunk 1 above)
Comment (Dartanboy) [reply src/main/java/com/battleasya/admin360/listener/JoinLeaveEvent.java:30]: Added all three suggestions.

Comment (denniemok) [reply src/main/java/com/battleasya/admin360/listener/JoinLeaveEvent.java:30]: Thanks! Looks great to me!

**Comment by denniemok** [comment]
Thanks for the contribution!

**Pull request merged**